crates/js-sys/src/lib.rs

```diff
@@ -28,6 +28,7 @@ use std::mem;
 use std::str;
 use std::str::FromStr;
 
+pub use wasm_bindgen;
 use wasm_bindgen::prelude::*;
```

Comment:
```suggestion
pub use wasm_bindgen::prelude::*;
```
I actually meant that we should re-export `wasm-bindgen` **in addition** to `JsValue`, but the prelude is even better!

Reply:
I think we should leave it like this.